remsfal-service/src/test/java/de/remsfal/service/boundary/ProjectResourceTest.java

```diff
@@ -378,7 +415,115 @@ void addProjectMemeber_SUCCESS_multiUser() {
+                .setParameter("projectTitle", TestData.PROJECT_TITLE_5)
+                .getSingleResult();
+        assertEquals(1, entries4);
+
+        long entries5 = entityManager
+                .createQuery("SELECT count(m) FROM ProjectEntity p JOIN p.memberships m WHERE p.title = :projectTitle", Long.class)
+                .setParameter("projectTitle", TestData.PROJECT_TITLE_5)
+                .getSingleResult();
+        assertEquals(1, entries5);
     }
 
+    @Test
+    void getProjectMembersTest(){
+        final String user1projectId1 = given()
+                .when()
+                .cookie(buildCookie(TestData.USER_ID_1, TestData.USER_EMAIL_1, Duration.ofMinutes(10)))
+                .contentType(MediaType.APPLICATION_JSON)
+                .body("{ \"title\":\"" + TestData.PROJECT_TITLE_1 + "\"}")
+                .post(BASE_PATH)
+                .then()
+                .statusCode(Status.CREATED.getStatusCode())
+                .extract().path("id");
+
+        given()
+                .when()
+                .cookie(buildCookie(TestData.USER_ID_1, TestData.USER_EMAIL_1, Duration.ofMinutes(10)))
+                .get(BASE_PATH + "/" + user1projectId1 + "/members")
+                .then()
+                .statusCode(Status.OK.getStatusCode())
+                .contentType(ContentType.JSON)
+                .body("members.id", Matchers.hasItem(TestData.USER_ID_1));
+    }
+
+    @Test
+    void updateProjectMemberTest() {
+        final String user1projectId1 = given()
+                .when()
+                .cookie(buildCookie(TestData.USER_ID_1, TestData.USER_EMAIL_1, Duration.ofMinutes(10)))
+                .contentType(MediaType.APPLICATION_JSON)
+                .body("{ \"title\":\"" + TestData.PROJECT_TITLE_1 + "\"}")
+                .post(BASE_PATH)
+                .then()
+                .statusCode(Status.CREATED.getStatusCode())
+                .extract().path("id");
+
+        given()
+                .when()
+                .cookie(buildCookie(TestData.USER_ID, TestData.USER_EMAIL, Duration.ofMinutes(10)))
+                .contentType(MediaType.APPLICATION_JSON)
+                .body("{ \"projectId\":\" " + user1projectId1 + "\",  \"id\":\"" + TestData.USER_ID_1 + "\", \"role\":\"LESSOR\", \"email\":\"max.mustermann@example.org\", \"title\":\"" + TestData.PROJECT_TITLE_1 + "\"}")
+                .patch(BASE_PATH + "/" + user1projectId1 + "/members/" + TestData.USER_ID_1)
+                .then()
+                .statusCode(Status.OK.getStatusCode())
+                .contentType(ContentType.JSON)
+                .and().body("id", Matchers.equalTo(user1projectId1))
+                .and().body("title", Matchers.equalTo(TestData.PROJECT_TITLE_1))
+                .and().body("members[0].role", Matchers.equalTo("LESSOR"));
+    }
+
+    @Test
+    void deleteProjectMemberTest(){
```

Comment:
This method name indicated that the following HTTP operation is tested... 
https://github.com/remsfal/remsfal-backend/blob/277c845828414a85acec1c687a57943c30c6126c/remsfal-core/src/main/java/de/remsfal/core/api/ProjectEndpoint.java#L134

Sonar says: https://sonarcloud.io/code?id=remsfal_remsfal-backend&pullRequest=158&selected=remsfal_remsfal-backend%3Aremsfal-service%2Fsrc%2Fmain%2Fjava%2Fde%2Fremsfal%2Fservice%2Fboundary%2FProjectResource.java&line=118

No HTTP DELETE method is called in this test. Here is something expected like given().....delete(BASE_PATH + "/members/" + TestData.User_ID_x)....